<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>142</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сканер</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string>Приложите палец</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
